src/index.js

```diff
@@ -587,55 +586,14 @@ export default class Bynder {
     return this.api.send('GET', 'v4/brands/');
   }
 
-  /**
-   * Gets the closest Amazon S3 bucket location to upload to.
-   * @see {@link https://bynder.docs.apiary.io/#reference/upload-assets/1-get-closest-amazons3-upload-endpoint/get-closest-amazons3-upload-endpoint}
-   * @return {Promise} Amazon S3 location url string.
-   */
-  getClosestUploadEndpoint() {
-    return this.api.send('GET', 'upload/endpoint');
-  }
-
-  /**
-   * Starts the upload process. Registers a file upload with Bynder and returns authorisation information to allow
-   * uploading to the Amazon S3 bucket-endpoint.
-   * @see {@link https://bynder.docs.apiary.io/#reference/upload-assets/2-initialise-upload/initialise-upload}
-   * @param {String} filename - filename
-   * @return {Promise} Relevant S3 file information, necessary for the file upload.
-   */
-  initUpload(filename) {
-    if (!filename) {
-      return rejectValidation('upload', 'filename');
-    }
-    return this.api.send('POST', 'upload/init', { filename });
-  }
-
-  /**
-   * Registers a temporary chunk in Bynder.
-   * @see {@link https://bynder.docs.apiary.io/#reference/upload-assets/3-upload-file-in-chunks-and-register-every-uploaded-chunk/register-uploaded-chunk}
-   * @param {Object} init - result from init upload
-   * @param {Number} chunkNumber - chunk number
-   * @return {Promise}
-   */
-  registerChunk(init, chunkNumber) {
-    const { s3file, s3_filename: filename } = init;
-    const { uploadid, targetid } = s3file;
-    return this.api.send('POST', 'v4/upload/', {
-      id: uploadid,
-      targetid,
-      filename: `${filename}/p${chunkNumber}`,
-      chunkNumber
-    });
-  }
-
   /**
    * Finalises the file upload when all chunks finished uploading and registers it in Bynder.
    * @see {@link https://bynder.docs.apiary.io/#reference/upload-assets/4-finalise-a-completely-uploaded-file/finalise-a-completely-uploaded-file}
-   * @param {String} fileId - Unique file identifier
-   * @param {String} fileName - Original file name
-   * @param {Number} chunksCount - Number of chunks
-   * * @param {Number} fileSize - File byte size
-   * @return {Promise}
+   * @param {String} fileId Unique file identifier
+   * @param {String} fileName Original file name
+   * @param {Number} chunksCount Number of chunks
+   * @param {Number} fileSize File byte size
+   * @return {Promise<object>} Object containing the correlation ID (`correlationId`) and file ID (`fileId`) of the upload
    */
   finaliseUpload(fileId, fileName, chunksCount, fileSize) {
     return this.api.send('POST', `v7/file_cmds/upload/${fileId}/finalise`, {
```

Comment:
Also this might need changing. This api call also needs `sha256` (as hexdigest)
`intent`. See here: https://bynder.atlassian.net/wiki/spaces/BAR/pages/776339703/File+service+interface+contract